SPRINTUSER_README.md

```diff
@@ -50,19 +50,25 @@ Find the Docker installer for your Operating System underneath the **installs**
  MacOS | Docker CE | Docker.dmg
  Linux | Docker CE, docker-compose | See [Linux instructions](https://docs.docker.com/install/#docker-ce)
 
-**⚠️ Docker Toolbox users**: Please install "Git for Windows" and "VirtualBox" as needed during the install process.
+**⚠️Docker Toolbox users**: Please click the checkboxes to install "Git for Windows" and "VirtualBox" as needed during the install process.
 
-**⚠️ Docker CE only** Start Docker
+**⚠️Docker Toolbox Users must start "Docker Quickstart Terminal" to start docker running.
```

Comment:
I initially didn't have this because it happens for these users in step 4 opening a terminal window, but it doesn't hurt.

Reply:
Missing a corresponding `**` after "Users" to terminate the bold otherwise the line begins with a literal `**`.

Reply:
Fixed.